.travis.yml

```diff
@@ -1,25 +1,30 @@
 language: elixir
 elixir:
-  - 1.6.6
```

Comment:
I think it's fine to keep 1.6.6 here, especially since it's still receiving security fixes.

Reply:
I'm alright with that, I think was a decision based on @NobbZ comment to test against last three major versions.